src/MDMMotionAnimator.m

```diff
@@ -125,7 +125,10 @@ - (void)animateWithTiming:(MDMMotionTiming)timing
     }
   }
 
+  [CATransaction begin];
```

Comment:
Line 123: Shouldn't the completion block be attached to the new CATransaction?

Reply:
The completion block is attached to the explicit animation up above.

Reply:
Ack.